templates/csharp/non-sso-tab-ssr/Components/Pages/TabConfig.razor

```diff
@@ -22,8 +22,8 @@
             {
                 SuggestedDisplayName = "My Tab",
                 EntityId = _entityId.ToString(),
-                ContentUrl = $"{NavigationManager.BaseUri}/tab",
-                WebsiteUrl = $"{NavigationManager.BaseUri}/tab"
+                ContentUrl = $"{NavigationManager.BaseUri}tab",
```

Comment:
Could you try the Uri class to combine the basic uri and path?
https://learn.microsoft.com/en-us/dotnet/api/system.uri.-ctor?view=net-8.0#system-uri-ctor(system-uri-system-string)

Reply:
Sure.